app/src-gen/fr/gaulupeau/apps/Poche/entity/ArticleDao.java

```diff
@@ -125,14 +125,14 @@ protected void bindValues(SQLiteStatement stmt, Article entity) {
     /** @inheritdoc */
     @Override
     public Long readKey(Cursor cursor, int offset) {
-        return cursor.isNull(offset + 0) ? null : cursor.getLong(offset + 0);
+        return cursor.isNull(offset) ? null : cursor.getLong(offset);
```

Comment:
To correct minor issues in auto-genereted code (more importantly, in the code that is likely be auto-regerenated again) is to waste time.